packages/infra/lib/ihe-stack/ihe-gateway.ts

```diff
@@ -84,20 +85,18 @@ export function createIHEGateway(stack: Construct, props: IHEGatewayProps): void
   const documentRetrievalPort = config.inboundPorts.documentRetrieval ?? documentQueryPort;
   const buildInboundAddress = (port: number) => `http://${iheGWAddressInbound}:${port}`;
 
-  const patientDiscoveryResource = apiResource.addResource("patient-discovery");
-  proxyToServer(patientDiscoveryResource, buildInboundAddress(patientDiscoveryPort));
-
-  const documentQueryResource = apiResource.addResource("document-query");
-  proxyToServer(documentQueryResource, buildInboundAddress(documentQueryPort));
-
-  const documentRetrievalResource = apiResource.addResource("document-retrieval");
-  proxyToServer(documentRetrievalResource, buildInboundAddress(documentRetrievalPort));
+  addProxyRoute("/v1/patient-discovery", buildInboundAddress(patientDiscoveryPort), apiGateway);
+  addProxyRoute("/v1/document-query", buildInboundAddress(documentQueryPort), apiGateway);
+  addProxyRoute("/v1/document-retrieval", buildInboundAddress(documentRetrievalPort), apiGateway);
```

Comment:
why did you remove `const apiRootV1 = api.root.addResource("v1");` in favor of putting `/v1` in every URL?

why not just build there from the `apiRootV1`?

Reply:
APIGatewayV2 doesnt support ".root" or the type that apiRoot was (an apig.IResource). This was the only way I could figure out to from the aws docs